rules_python/whl.py

```diff
@@ -126,19 +140,33 @@ def main():
 load("{requirements}", "requirement")
 
 py_library(
-  name = "pkg",
-  srcs = glob(["**/*.py"]),
-  data = glob(["**/*"], exclude=["**/*.py", "**/* *", "BUILD", "WORKSPACE"]),
-  # This makes this directory a top-level in the python import
-  # search path for anything that depends on this.
-  imports = ["."],
-  deps = [{dependencies}],
-     )""".format(
-       requirements=args.requirements,
-       dependencies=','.join([
-         'requirement("%s")' % d
-         for d in whl.dependencies()
-       ])))
+    name = "pkg",
```

Comment:
Is this an intentional change from 2 to 4 space indent, because it goes into a BUILD file?

Side note: I indent everything open-source with 4 spaces, instead of the Google 2, except when I forget.

Reply:
It's been a while, so let's say yes :)

Yeah, the external emacs mode screws with me.  I didn't realize the world didn't do 2 space.